ginga/examples/configs/channel_Image.cfg

```diff
@@ -0,0 +1,118 @@
+#
+# Image channel preferences file
+#
+# Place this in file under ~/.ginga with the name "channel_Image.cfg" for
+# channel name "Image". This file WILL BE OVERWRITTEN by the Preferences
+# plugin (see below). NOT ALL PARAMETERS IN GUI CAN BE ADJUSTED HERE.
+#
+# Alternately, the settings can be adjusted via Preferences local plugin
+# within Ginga. NOT ALL PARAMETERS CAN BE ADJUSTED VIA GUI but users can
+# normally ignore the hidden ones.
+#
+
+# ---------------
+# Color Distribution
+
+color_algorithm = 'linear'
+
+# ---------------
+# Color Mapping
+
+color_map = 'gray'
+color_hashsize = 65535
+intensity_map = 'ramp'
+
+# ---------------
+# Auto Cuts
+
+autocut_method = 'zscale'
+autocut_params = {}
+cuts = (0.0, 0.0)
+
+# ---------------
+# Transform
+
+flip_x = False
+flip_y = False
+swap_xy = False
+rot_deg = 0.0
+
+# ---------------
+# WCS
+
+wcs_coords = 'icrs'
+wcs_display = 'sexagesimal'
+
+# ---------------
+# Zoom
+
+zoom_algorithm = 'step'
+zoom_rate = 1.4142135623730951
+zoomlevel = 1.0
+scale_x_base = 1.0
+scale_y_base = 1.0
+scale = (1.0, 1.0)
+scale_min = 1e-05
+scale_max = 10000.0
+interpolation = 'basic'
+
+# ---------------
+# Panning
+
+pan = (1.0, 1.0)
+pan_coord = 'data'
+show_pan_position = False
+
+# ---------------
+# New Images
+
+autocuts = 'override'
+autozoom = 'on'
+autocenter = 'on'
+switchnew = True
+raisenew = True
+genthumb = True
+
+# ---------------
+# General - Inherits from general.cfg if not specified, otherwise overwrite.
+
+# Number of images to keep in memory per channel (0 = unlimited)
+# Same as numImages in general.cfg
+numImages = 10
+
+# Channel focus follows window focus
+# Same as channelFollowsFocus in general.cfg???
+follow_focus = True
```

Comment:
If `follow_focus` is True then the channel will be changed when the mouse cursor enters the channel window.  I'm pretty sure this is passed on from the `general.cfg` setting for `channel_follows_focus`.  You won't actually notice this effect unless you use gridded workspaces or alternate viewer layouts, because clicking a tab also changes the focus.